app/assets/stylesheets/components/navbar.scss

```diff
@@ -53,7 +53,6 @@
       margin-top: 37.8px;
       width: 30px;
       height: 30px;
-      @include border-radius(50%);
```

Comment:
Why was this removed @renziver? its needed to style the users avatar correctly on the navbar. Can we put it back in please?